README.md

```diff
@@ -1,2 +1,7 @@
 # holiday-hackathon-bot
 The bot for the end-of-year hackathon &amp; week of workshops organized by the Foothill CS Club.
+
+```shell
+# User "holiday" with the password "placeholder"
+docker run --name holiday -e POSTGRES_PASSWORD=placeholder -d postgres
+```
```

Comment:
What's this for?

Reply:
So you can set the password for the database, postgres doesn't like when there's no password.